README.md

```diff
@@ -809,6 +809,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Geek-Jokes](https://github.com/sameerkumar18/geek-joke-api) | Fetch a random geeky/programming related joke for use in all sorts of applications | No | Yes | Yes |
 | [Genshin Impact](https://genshin.dev) | Genshin Impact game data | No | Yes | Yes |
 | [Giant Bomb](https://www.giantbomb.com/api/documentation) | Video Games | `apiKey` | Yes | Unknown |
+| [GraphQL Pokemon](https://graphqlpokemon.favware.tech) | GraphQL powered Pokemon API. Supports generations 1 through 8 | No | Yes | Yes |
```

Comment:
I believe the link to the official repository is better: https://github.com/favware/graphql-pokemon

It contains more information about the project.

```suggestion
| [GraphQL Pokemon](https://github.com/favware/graphql-pokemon) | GraphQL powered Pokemon API. Supports generations 1 through 8 | No | Yes | Yes |
```

Reply:
Ah yeah, that's fair. I wasn't sure which of the 2 to put there.